update_action.py

```diff
@@ -0,0 +1,42 @@
+"""
+  Scans directories for files that satisfy a specific mask and updates
+  the actions' tags.
+  python3 update_action.py --input “one/*.y*ml two/*.y*ml three/*.y*ml” --name mlflow
+"""
+
+import argparse
+import glob
+import re
+from github import Github
+from pathlib import Path
+
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("-i", "--inputs", type=str, help="Input directories to scan")
+
+parser.add_argument("-n", "--name", type=str, help="Name of the action")
```

Comment:
Why not all actions?

We might detect the workflow file by its `kind:`, or any other neuro-flow specific attribute and the action by `gh:` or `github:` prefix.
Otherwise, if we have 5 actions, we will have 5 launches of this script. Not a critical problem, but as for me a single launch is more attractive and comfy.